test/src/sdk-harness/test_projects/auth/mod.rs

```diff
@@ -94,3 +99,109 @@ async fn get_contracts() -> (
         wallet.lock(),
     )
 }
+
+#[tokio::test]
+async fn can_get_predicate_id() {
+    // Setup Wallets
+    let asset_id = AssetId::default();
+    let wallets_config = WalletsConfig::new_multiple_assets(
+        2,
+        vec![AssetConfig {
+            id: asset_id,
+            num_coins: 1,
+            coin_amount: 1_000,
+        }],
+    );
+    let wallets = &launch_custom_provider_and_get_wallets(wallets_config, None, None).await.unwrap();
+    let first_wallet = &wallets[0];
+    let second_wallet = &wallets[1];
+
+    // Setup Predciate
+    let hex_predicate_address: &str = "0x935a191561e0e9388c1b5dfc4626f6ecd98b2b3dd416a9a9e9ce2f0bb214f4b8";
```

Comment:
Is this a hardcoded predicate id? Can you make it calculate the id within the test? Its annoying to have to change the contract addresses in many of the tests everytime whenever the bytecode changes, dont wanna add to that

Reply:
Not possible as we need to encode and pass it as data on line 126.